be/extmap.c

```diff
@@ -178,9 +184,14 @@ static void emap_key_init(struct m0_be_emap_key *key)
 static void emap_rec_init(struct m0_be_emap_rec *rec)
 {
 	m0_format_header_pack(&rec->er_header, &(struct m0_format_tag){
-		.ot_version = M0_BE_EMAP_REC_FORMAT_VERSION,
-		.ot_type    = M0_FORMAT_TYPE_BE_EMAP_REC,
-		.ot_footer_offset = offsetof(struct m0_be_emap_rec, er_footer)
+		        .ot_version = M0_BE_EMAP_REC_FORMAT_VERSION,
```

Comment:
[* style *] use tabs at beginning of lines instead of spaces.

Reply:
Fixed.